src/index.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 import ReactDOM from 'react-dom/client';
+import { PersistGate } from 'redux-persist/integration/react';
 
 import './index.css';
```

Comment:
This is grouped separaely since it's a relative import.

Reply:
done